merp_outgoing_routing/tests/__init__.py

```diff
@@ -0,0 +1,3 @@
+# -*- coding: utf-8 -*-
```

Comment:
This is no longer needed. You can use pylint-odoo tool to validate code.

Please use creds as in other files:
Copyright 2019 VentorTech OU
Part of Ventor modules. See LICENSE file for full copyright and licensing details.